Application/People/Meta/Student/Service/Data.php

```diff
@@ -378,17 +459,27 @@ public function createStudentMedicalRecord(
         $Medication,
         $AttendingDoctor,
         $InsuranceState,
-        $Insurance
+        $Insurance,
+        $MasernDate = null,
+        TblStudentMasernInfo $MasernDocumentType = null,
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$MasernDocumentType" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)